src/bandersnatch/verify.py

```diff
@@ -137,7 +145,14 @@ async def verify(
                     all_package_files.append(pkg_file)
                     continue
                 else:
-                    await master.url_fetch(jpkg["url"], pkg_file, executor)
+                    try:
+                        await master.url_fetch(jpkg["url"], pkg_file, executor)
+                    except Exception:
```

Comment:
Again only really expecting http timeouts (sure we could get OS Errors) ...
```suggestion
                    except aiohttp.ClientError:
```

Reply:
Same as above.